README.md

```diff
@@ -42,18 +40,12 @@ Juste use `node index.js` to start everything. You should create a file called `
 
 ## Commands
 
-6brain not only pushed the data, it can respond to sms stimuli:
-
-- `status` send back a message with the current status
-- `ip` sends back the ip
-- `openTunnel:2222:9632:kerrigan` opens a reverse ssh tunnel toward `kerrigan` (must be set in `~/.ssh/config`) and send a message once the tunnel is up.
+6brain not only push datas, it can respond to sms and MQTT stimuli:
```

Comment:
Full list of commands will be soon available under this link: https://github.com/anthill/pheromon/api/clients/Admin/ReadMe.md